### ISSUE TITLE
Dual Clutch Paddle Functionality

### ISSUE THREAD

**Issue by SrgntBallistic** [Snoothy/UCR#243]
Hello,

In sim racing some wheels have firmware functionality that combines 2 analog axes into 1 axis. This is done in a way that a "Bitepoint" is specified such that one of the clutches ("slave clutch") goes from 0% to the Bitepoint % (ex: 0-50%).

The other clutch ('master clutch") is set to go from 0 to 100%.

As such a sim racer can pull in both clutches. Then release the master clutch with the slave clutch still pulled to it's maximum. The resulting axis value would be right at the bitepoint which gives the car it's best launch. Ideally the slave clutch is then control from the bitepoint to 0% to minimize wheel spin or bogging/stalling.

However, some wheels don't have this functionality built in and the clutches just map to 2 separate axes

I've been trying a way to utilize existing functionality in UCR to achieve the desired bitepoint with 2 separate physical input axes.

I think it could be achievable if the Axis to Axis allowed a max value. So:

Map Physical Clutch 1 Axis-to-Axis as Virtual Axis 1, from 0 to Bite point % as the max
Map Physical Clutch 2 Axis-to-Axis as Virtual Axis 2, from 0 to 100% as the max

Axis Merger, Virtual Axis 1 and Virtual Axis 2 with Greatest Mode resulting in Virtual Axis 3

Could functionality as I've described above, or perhaps some more direct functionality be added to the project?

Is there a simple way to achieve this with current functionality that I haven't thought of yet?

Thanks

**Comment by lewisst** [comment]
@SrgntBallistic have you seen Dan Suzuki video?[ iRacing Guides | How to do insane standing starts! No Clutch needed!](
https://youtu.be/PHvve_GoEz0?t=413)

I've dual paddles haven't attempted it, but he uses UCR to map **button to axis** value to create a dual clutch.

**Comment by SrgntBallistic** [comment]
@lewisst Yes I've seen Dan's video. I frequent his stream and Discord a lot.

That specific method of launching the Porsche Cup car doesn't work any longer. iRacing added clutch damage to it's model so just holding the brake with the clutch at the bite point is actually detrimental now. Also I'd consider the button method a bit less ideal since it simply drops the clutch to a specific value instead of allowing you to modulate it with analog inputs.

Using the clutch paddles in the method I mentioned works universally for pretty much every car without damaging the clutch. And since a lot of people have clutch paddles on their wheels but no way to set a bite point or combine them in this specific way

**Comment by techguytom** [comment]
I came here hoping to find a solution to this as well. We could achieve the same thing with a single axis to axis if the same options were available that are present in the button to axis settings. A max value so when you roll off the clutch the zero point is the same.

**Comment by schraal** [comment]
Same here. I have it working like so:
- axis to axis mapping
- Fanatec clutch paddle (left or right) as source
- vJoy axis as output (like in the Dan Suzuki video)
- Sensitivity - percentage = 42
- Sensitivity - linear = true
This almost works perfectly, except if I pull in the paddle too quick the value does not go to 42%, but to a lower value.

Update: Can't get it working stably. After a reboot it does not work anymore. And sometimes it just takes the paddle input directly instead of the mapped vjoy device. Don't understand why yet
